lib/unifex/code_generator/base_types/string.ex

```diff
@@ -1,6 +1,6 @@
 defmodule Unifex.CodeGenerator.BaseTypes.String do
   @moduledoc """
-  Module implementing `Unifex.CodeGenerator.BaseType` behaviour for Unifex state.
+  Module implementing `Unifex.CodeGenerator.BaseType` behaviour for strings.
```

Comment:
I think the docs for all BaseTypes should mention into what native type they are translated. In this case it is important that Elixir strings become NULL-terminated C-strings